src/hooks/useNYPLBreakpoints.ts

```diff
@@ -0,0 +1,29 @@
+import { useMediaQuery } from "@chakra-ui/react";
+
+/**
+ * This hook is used to determine if the current screen size is larger than
+ * the specific NYPL breakpoint values. The returned value is an object with
+ * boolean values for each breakpoint.
+ */
+const useNYPLBreakpoints = () => {
+  const [
+    isLargerThanSmall,
+    isLargerThanMedium,
```

Comment:
Echoing my comment from `ButtonGroup.tsx`, would it make sense to also include something like isLargerThanMobile in addition to these four base options?